sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Queues/src/Microsoft.Azure.WebJobs.Extensions.Storage.Queues.csproj

```diff
@@ -6,6 +6,7 @@
     <Version>5.2.0-beta.1</Version>
     <!--The ApiCompatVersion is managed automatically and should not generally be modified manually.-->
     <ApiCompatVersion>5.1.3</ApiCompatVersion>
+    <RunApiCompat>false</RunApiCompat>
```

Comment:
I'm not sure I have enough context to understand how this would pass. Is the output not included in the build from elsewhere so as to not trigger the error above? If so, 👍🏻

Reply:
This should fail - I am just verifying that it does and then I will revert it.